src/Microsoft.AspNetCore.ResponseCaching/Internal/ResponseCacheContext.cs

```diff
@@ -37,6 +40,8 @@ public class ResponseCacheContext
 
         public CachedVaryByRules CachedVaryByRules { get; internal set; }
 
+        internal ILogger Logger { get; }
```

Comment:
why not `public`? after all the context itself is public which means it is open to application consumption.